src/site/content/en/blog/advancing-framework-ecosystem-cds-2019/index.md

```diff
@@ -0,0 +1,208 @@
+  </figcaption>
+</figure>
+
+Bundle size reduction for angular.io with differential builds
+(from[" Version 8 of Angular"](https://blog.angular.io/version-8-of-angular-smaller-bundles-cli-apis-and-alignment-with-the-ecosystem-af0261112a27))
+
++   Support for standard dynamic import syntax for lazy-loading routes.
++   [Web worker support](https://angular.io/guide/web-worker).
++   [Ivy](https://www.youtube.com/watch?v=jnp_ny4SOQE&feature=youtu.be&t=1320), Angular's new
+    rendering engine, is available in [preview
+    mode](https://angular.io/guide/ivy#opting-into-angular-ivy) for existing projects.
+
+You can learn more about these improvements in
+["Version 8 of Angular"](https://blog.angular.io/version-8-of-angular-smaller-bundles-cli-apis-and-alignment-with-the-ecosystem-af0261112a27)
+and the Chrome team looks forward to working with them closely in the next year as more features
+land.
+
+## Next.js
+
+[Next.js](https://nextjs.org/) is a web framework that uses React as a view layer. In addition to a
+UI component model that many developers expect from a client-side framework, Next.js provides a
+number of built-in default features:
+
++   Routing with default code-splitting
++   Compilation and bundling (using [Babel](https://babeljs.io/) and
+    [webpack](https://webpack.js.org/))
++   Server-side rendering
++   Mechanisms to fetch data at a per-page level
++   Encapsulated styling (with [styled-jsx](https://github.com/zeit/styled-jsx))
+
+Next.js optimizes for reduced bundle sizes, and the Chrome team helped identify areas where we could
+help further improve performance. You can learn more about each of them by viewing their requests
+for comments (RFCs) and pull requests (PRs):
+
+1.  An improved webpack chunking strategy that emits more granular bundles, reducing the
+    amount of duplicate code fetched through multiple routes
+    ([RFC](https://github.com/zeit/next.js/issues/7631),
+    [PR](https://github.com/zeit/next.js/pull/7696)).
+2.  Differential loading with the
+    [module/nomodule pattern](../serve-modern-code-to-modern-browsers/#use-lessscript-type%22module%22greater)
+    which can reduce the total amount of JavaScript in Next.js apps by up to 20% with no code
+    changes ([RFC](https://github.com/zeit/next.js/issues/7563),
+    [PR](https://github.com/zeit/next.js/pull/7704)).
+3.  Improved performance metric tracking which utilizes the User Timing API
+    ([PR](https://github.com/zeit/next.js/pull/8069)).
+
+<figure class="w-figure w-figure--center">
+  <img src="barnebys.png" class="w-screenshot" alt="Homepage of Barnebys.com">
+  <figcaption class="w-figcaption">
+    <a href="https://www.barnebys.com/">Barnebys.com</a>, a large search engine for antiques and collectibles, saw a 23% reduction in total JavaScript with granular chunking enabled
```

Comment:
```suggestion
    <a href="https://www.barnebys.com/">Barnebys.com</a>, a large search engine for antiques and collectibles, saw a 23% reduction in total JavaScript after enabling granular chunking
```

Rephrasing this sentence slightly to emphasize the action that created the improvement (this is kind of a nit but please just trust my writer intuition on this)